firebase.rules.json

```diff
@@ -22,6 +22,10 @@
     // Anyone may read content
     "content": {
       ".read": true
+    },
+    "suggestions":  {
```

Comment:
Looks good. Beware that I might have made some change to the upstream rules without doing it hte Right Way through this file, we should check before we deploy.